Files/ViewModels/SettingsViewModel.cs

```diff
@@ -760,6 +769,12 @@ public SortOption DefaultDirectorySortOption
             set => Set((byte)value);
         }
 
+        public bool MultiselectEnabled
+        {
+            get => Get(false);
+            set => Set(value);
+        }
```

Comment:
Should this be part of the SettingsViewModel?

Reply:
@yaichenbaum I think it should be. I had trouble getting a "enable multiselect by default" option working reliably, so the multiselect option is persistent instead.

Reply:
That's fine, my question was regarding if this is the correct place to declare the variable, shouldn't this be in a different view model?

Reply:
@yaichenbaum I thought it was fine, since it allows the setting to be persistent and is accessible to both the status center and the layouts, but I could be wrong. Where do you think it should go?

Reply:
This setting should really be temporary since it's meant to toggle the selection option, I would recommend putting it in `InteractionViewModel`. Technically this should be called `MainViewModel` but that's beyond the scope of this PR.